Common/Scheduling/DateRules.cs

```diff
@@ -92,7 +92,7 @@ public IDateRule EveryDay()
         public IDateRule EveryDay(Symbol symbol)
         {
             var security = GetSecurity(symbol);
-            return new FuncDateRule(symbol.ToString() + ": EveryDay", (start, end) => Time.EachTradeableDay(security, start, end));
+            return new FuncDateRule(symbol.Value + ": EveryDay", (start, end) => Time.EachTradeableDay(security, start, end));
```

Comment:
I know outputting SID's isn't the prettiest thing, but a nice property is that they don't change. Scheduled events can be added but they can also be removed, and in order to removed them you need to know the name of the event. If we create the event and then some time later the symbol's ticker changes and we try to remove it we won't be able to find it.

Was this change purely for aesthetics?

Reply:
When possible, especially in logging, I often replace `symbol.ToString()` with `symbol.Value`.

In this case, I thought it wouldn't be an issue: 
removing an event with `ScheduleManager.Remove` from an algorithm only requires passing `scheduledEvent.Name` (returned by `ScheduleManager.On`), instead of recreating the same name generated when the event was created.

Reply:
I suppose it's not a big issue since we do return the scheduled event instance, so if they wanted to remove it later they could save off the instance or the name when they create it.

Reply:
I would recommend against updating logs to use `symbol.Value` -- for a pretty good reason why, check out `UniverseSelectionRegressionAlgorithm` -- where we have multiple GOOGs back to back that are different securities. Without the full identifier, it can be hard to tell what is what.